src/main/java/com/gmail/woosay333/onlinebookstore/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package com.gmail.woosay333.onlinebookstore.exception;
+
+public class RegistrationException extends RuntimeException {
```

Comment:
Actually I think it was part of the task and requirements implied that we should have this Ex as checked:
> Add AuthenticationController with the following methods:
public UserResponseDto register(UserRegistrationRequestDto request) throws RegistrationException